spec/kenji_spec.rb

```diff
@@ -204,9 +192,58 @@ def app; app_for('5', root_controller: Spec5::MainController); end
       last_response.status.should == 500
     end
   end
-  
-  # TODO: Write unit tests for :catch_exceptions option.
+
+  context '7' do
+    context 'catch_exceptions is false' do
+      def app
+        app_for('7',
+                catch_exceptions: false)
+
+        it 'should raise' do
+          -> { get '/main/crasher' }.must_raise
+        end
+      end
+    end
+
+    context 'catch_exceptions is true' do
+      context :exception_in_body do
+        context 'exception_in_body is false' do
+          def app
+            app_for('7',
+                    catch_exceptions:  true,
+                    exception_in_body: false)
+          end
+
+          it 'should return 500 for exceptions' do
+            get '/main/crasher'
+            last_response
+              .body
+              .should == {status: 500,
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Space inside { missing.